digital-deck-server/models/player.js

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
This file and `table.js` are just moved from `/services` to `/models` right? Funny how it shows that for `session.js` and not these 2.